ui_framework/src/global_styling/reset/_reset.scss

```diff
@@ -34,10 +34,44 @@ footer, header, hgroup, menu, nav, section {
   display: block;
 }
 
+html {
+  font-family: $kuiFontFamily;
+  font-weight: $kuiFontWeightRegular;
+  color: $kuiColorDarkestGray;
```

Comment:
We might want to alias this behind something like `$kuiBodyFontColor` so that if someone messes around with the palette in a theme, they can directly control this base color.